nix/overlays.nix

```diff
@@ -21,6 +21,7 @@
       funflow = project.funflow.components.library.overrideAttrs (old:
         { buildInputs = old.buildInputs ++ [ super.docker ]; }
       );
+      funflow-examples = project.funflow-examples.components.library;
```

Comment:
Nice the updates to the nix packaging look good to me. The only thing I wonder is whether we need to expose this librry or just the `makefile-tool` executable.